doc/dev/installation.rst

```diff
@@ -81,6 +81,21 @@ The essential steps are:
 
    .. group-tab:: Linux Debian/Ubuntu
 
+      .. warning::
+
+        If using Anaconda or Miniconda, please make sure to upgrade ``libstdcxx-ng``:
+
+        .. code-block:: console
+
+          conda install -c conda-forge libstdcxx-ng
+
+        If not, you will receive the following error when running ``make test`` because the conda
```

Comment:
```suggestion
        If not, you may receive the following error when running ``make test`` because the conda
```

Depending on the system you build on, this could happen, but may not.